test/agentchat/test_compressible_agent.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import sys
+import autogen
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST
+
+try:
+    from autogen.agentchat.contrib.compressible_agent import (
+        CompressibleAgent,
+    )
+
+    skip_test = False
+except ImportError:
+    skip_test = True
+
+
+@pytest.mark.skipif(
+    sys.platform in ["darwin", "win32"] or skip_test,
+    reason="do not run on MacOS or windows",
+)
+def test_compressible_agent():
+    try:
+        import openai
+    except ImportError:
+        return
```

Comment:
this can probably be removed, `openai` isn't used in this test

Reply:
Sorry if this dumb questions, what is the reason for skipping this test on Mac OS?